### DISCUSSION TITLE
DB/db13/

### DISCUSSION THREAD

**Discussion by giscus** [HobbitQia/notebook#29] [General]
# DB/db13/

HobbitQia的笔记本

https://note.hobbitqia.cc/DB/db13/

**Comment by birchtree2** [comment]
Concurrent Executions一节里
unrepeatable read的配图似乎放错了（和dirty read重复），课件上应该是下面这张：
![image-20240511144741954](https://s2.loli.net/2024/05/11/KUZxCuBJeTOwShr.png)
